repository/Seaside-Tests-Functional.package/WAPerformanceFunctionalTest.class/instance/renderAnchorsOn..st

```diff
@@ -4,7 +4,7 @@ renderAnchorsOn: canvas
 		html unorderedList: [
 			html listItem: [
 					html anchor
-						name: 'name';
+						id: 'name';
```

Comment:
AFAIK this is rendered multiple times. Strictly speaking this leads to invalid HTML as we have multiple element with the same id. I'm fine with either of these:

- Leave as is, it's not a big deal.
- Make it unique.
- Switch to a different attribute like `title`

Reply:
You're right. Though not really important here, I switched to title.